<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>realease_task</class>
 <widget class="QDialog" name="realease_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>36</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>348</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>原文语言</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>目标语言</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>负责人报名截止日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>总薪酬</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>36</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>160</height>
    </size>
   </property>
   <property name="text">
    <string>简介</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>72</y>
     <width>250</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>348</width>
     <height>160</height>
    </size>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_orin">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>32</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>6</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最终期限</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>给予负责人的薪酬</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
